proto/ibc/applications/fee/v1/ack.proto

```diff
@@ -7,9 +7,11 @@ option go_package = "github.com/cosmos/ibc-go/v3/modules/apps/29-fee/types";
 import "gogoproto/gogo.proto";
 
 // IncentivizedAcknowledgement is the acknowledgement format to be used by applications wrapped in the fee middleware
-// It contains the raw acknowledgement bytes, as well as the forward relayer address
 message IncentivizedAcknowledgement {
-  bytes  result                  = 1;
+  // the acknowledgement result bytes
```

Comment:
This is actually the underlying app ack (we still need to update the field name)

Reply:
Aha! Good spot, I'll leave the field as is for now and it can be updated later. Would rather spare adding code changes on top of this. I'll update the doc comment with the suggestion below and regenerate docs 👍 

```suggestion
  // the underlying app acknowledgement result bytes
```